api/src/test/java/org/pmiops/workbench/api/NotebooksControllerTest.java

```diff
@@ -701,7 +361,41 @@ public void testNotebookLockingMetadataKnownUser() {
   }
 
   @Test
-  public void testNotebookLockingMetadataUnknownUser() {
+  public void testGetNotebookLockingMetadata_nullMetadata() {
+    final Map<String, String> gcsMetadata = null;
+
+    // This file has no metadata so the response is empty
+
+    final NotebookLockingMetadataResponse expectedResponse = new NotebookLockingMetadataResponse();
+    assertNotebookLockingMetadata(gcsMetadata, expectedResponse, fcWorkspaceAcl);
+  }
+
+  @Test
+  public void testGetNotebookLockingMetadata_plaintextUser() {
+    final String lastLockedUser = LOGGED_IN_USER_EMAIL;
+    final Long lockExpirationTime = Instant.now().plus(Duration.ofMinutes(1)).toEpochMilli();
+
+    final Map<String, String> gcsMetadata =
+        new ImmutableMap.Builder<String, String>()
+            .put(LOCK_EXPIRE_TIME_KEY, lockExpirationTime.toString())
+            // store directly in plaintext, to show that this does not work
+            .put(LAST_LOCKING_USER_KEY, lastLockedUser)
+            .put("extraMetadata", "is not a problem")
+            .build();
+
+    // in case of accidentally storing the user email in plaintext
+    // it can't be retrieved by this endpoint
+
+    final NotebookLockingMetadataResponse expectedResponse =
+        new NotebookLockingMetadataResponse()
+            .lockExpirationTime(lockExpirationTime)
+            .lastLockedBy("UNKNOWN");
```

Comment:
off topic for this PR, but this is an interesting case to think about for last-modified